cmd/autoconfigbrancher/helper/helper.go

```diff
@@ -0,0 +1,24 @@
+package helper
```

Comment:
Why does this need to be a separate package?

Reply:
I want to keep this open to other tools for running `bash` command.
I feel it is cleaner this way than using `bumper` from upstream.

Reply:
I think this package has very little utility as noted in other comments, we should consider removing it

Reply:
Removing ...

Reply:
Done.